src/main/java/com/gtnewhorizons/neid/mixins/early/minecraft/MixinS21PacketChunkData.java

```diff
@@ -0,0 +1,47 @@
+package com.gtnewhorizons.neid.mixins.early.minecraft;
+
+import net.minecraft.network.play.server.S21PacketChunkData;
+import net.minecraft.world.chunk.storage.ExtendedBlockStorage;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Constant;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.ModifyConstant;
+import org.spongepowered.asm.mixin.injection.Redirect;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import com.gtnewhorizons.neid.mixins.interfaces.IExtendedBlockStorageMixin;
+
+@Mixin(S21PacketChunkData.class)
+public class MixinS21PacketChunkData {
+
+    @Shadow
+    private static byte[] field_149286_i;
+
+    @Inject(method = "<init>()V", at = @At(value = "RETURN"), require = 1)
+    private void notenoughIDs$S21PacketChunkDataConstructorAddition(CallbackInfo CI) {
```

Comment:
can you remove the `S21PacketChunkData` from the method names in this whole class